deploy/ocp-templates/glusterfs-template.yaml

```diff
@@ -35,6 +35,12 @@ objects:
         - image: gluster/gluster-centos:latest
           imagePullPolicy: IfNotPresent
           name: glusterfs
+          QoS Tier:
+            memory: Guaranteed
```

Comment:
We expect this to change at some point to make Guaranteed more suitable for these types of workloads. But we think the Guaranteed tier is currently NOT what we should do.